tests/unittests/Foundation/ReferenceFoundation/TestNSArray.mm

```diff
@@ -150,13 +150,19 @@ static int objectIndexInArray(NSArray* array, int value, int startingFrom, int l
     //        NSBinarySearchingOptions* searchForBoth = NSBinarySearchingFirstEqual | NSBinarySearchingLastEqual
     //        auto _ = objectIndexInArray(array, 9, 0, 13, searchForBoth)
 
-    auto notFound = objectIndexInArray(array, 11, 0, 13);
-    ASSERT_EQ_MSG(notFound, NSNotFound, @"NSArray return NSNotFound if object is not found.");
```

Comment:
It looks like he said the same? [https://github.com/Microsoft/WinObjC/pull/557#discussion_r67045139](https://github.com/Microsoft/WinObjC/pull/557#discussion_r67045139)

---
In reply to: [67917856](https://github.com/Microsoft/WinObjC/pull/569#discussion_r67917856) [](ancestors = 67917856)